### DISCUSSION TITLE
将数据存储从LeanCloud国内版本迁移到图际版本后无法记录

### DISCUSSION THREAD

**Discussion by elkan1788** [walinejs/waline#1051] [Q&A] [answered]
之前一直都在使用 LeanCloud 国内的版本，发现评论的速度确实太慢了点，于是尝试创建了个 LeanCloud 国际版本，然后也更新了环境的如下相关变量参数：

![image](https://user-images.githubusercontent.com/1217096/169770760-c1bcdaf5-b592-4ed0-8df9-ea35a2c33b8b.png)

接着将数据从国内数据存储导入到了国际版本的数据存储中，加载评论的速度确实是有所提升，但发现无法记录新发的评论内容，通过 F12 调试也没有发现报错，但后台就是没新的记录出现，这是为何呢？

权限开放了所有：

![0@8$BC`BK%DJ3FOJRX9W9T9](https://user-images.githubusercontent.com/1217096/169771139-7faf767f-5814-4938-9b29-d7b50b0f288a.png)

新评论的邮件通知也收到：

![screenshot-20220523-154834](https://user-images.githubusercontent.com/1217096/169771337-d7a8128f-b717-45f2-8b26-6419b2249c52.png)

但只要一刷新业务，评论就不显示了，后台查看也是没有记录数的。

**Comment by lizheming** [comment, marked as answer]
看下是不是进审核或者垃圾评论了

**Reply by elkan1788** [reply]
确认过了，并没有进入到审核和垃圾评论里（直接看Class对象都是没有新的记录产生的）

**Reply by lizheming** [reply]
更新完环境变量需要 redeploy 一下，否则还是老的环境变量

**Reply by elkan1788** [reply]
是的，每次修改变量都有重新 redeploy 操作的，都检查过有调用API的域名和路径是有变化的。现暂不清楚什么情况，只能是先退回到同内的版本，然后评论和管理功能都恢复正常！
